### PR TITLE
Use "New Module Imports"

### PR DESCRIPTION
**Pull request by Turbo87**
see https://github.com/ember-cli/ember-rfc176-data

### CODE REVIEW

**Comment by Turbo87** [comment]
CI failure seems unrelated 😞

**Comment by jasonmit** [comment]
@Turbo87 I'll need to dig into all these CI failures across the addons I maintain - unsure what changed upstream.  At the least I should be allow beta and canary to fail.

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "chalk": "^1.1.3",
     "cldr-core": "28.0.0",
     "ember-assign-polyfill": "^2.0.1",
-    "ember-cli-babel": "^6.0.0",
+    "ember-cli-babel": "^6.6.0",
```
Comment (jasonmit) [inline package.json]: @Turbo87 I suspect the polyfill is occurring here

Comment (jasonmit) [inline package.json]: I confirmed this, disregard.  https://github.com/babel/ember-cli-babel/blob/master/package.json#L39

**Comment by jasonmit** [comment]
@Turbo87 LGTM.  I will version a minor soon

**Comment by Turbo87** [comment]
@jasonmit I actually was just made aware that I introduced a bug here since the new imports are not available in the app folder. Will fix tomorrow :(

**Pull request merged**